src/components/LocaleChanger.vue

```diff
@@ -29,6 +29,8 @@ export default {
           return require("../assets/lang_germany.svg");
         case "da":
           return require("../assets/lang_denmark.svg");
+        case "tr":
```

Comment:
In order to make turkish available in the selection, you need to add it to the components `data` (line 23).

Reply:
@tlahmann  currently the flag is to big and vue does not scale it automatically. The flag .svg is added already. Can you re-size it ?